tests/utils.go

```diff
@@ -762,3 +763,12 @@ func NewConsoleExpecter(virtCli kubecli.KubevirtClient, vm *v1.VirtualMachine, c
 		Check: func() bool { return true },
 	}, timeout, opts...)
 }
+
+// SkipBecauseOfVersion will skip the test if current k8s version is less than expected one
+func SkipBecauseOfVersion(version string, reason string) {
+	if v, exists := os.LookupEnv("KUBE_VERSION"); exists {
```

Comment:
Would you mind directly querying the k8s version REST endpoint inside go?

Reply:
@rmohr Do you have an example, I tried to find how can I do it and without result :disappointed:

Reply:
Sure:

```golang
kubecli.CoreV1().RESTClient().Get().AbsPath("/version").Do().Raw()
```

Reply:
btw running a kubectl command with `-v 9` attached is a nice trick to see what it is querying.